docs/v1/index.bs

```diff
@@ -3040,37 +3043,41 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       1. Let |requests| be an empty [=list=].
       1. Let |responses| be an empty [=list=].
       1. Let |resultList| be an empty [=list=].
-      1. If |options|.{{CacheQueryOptions/ignoreMethod}} is false and |request|'s [=request/method=] is not \`<code>GET</code>\`, return |resultList|.
-      1. Let |cachedURL| and |requestURL| be null.
       1. Let |storage| be null.
       1. If the optional argument |targetStorage| is omitted, set |storage| to the [=relevant request response list=].
       1. Else, set |storage| to |targetStorage|.
       1. [=list/For each=] |requestResponse| of |storage|:
-          1. Set |cachedURL| to |requestResponse|'s request's [=request/url=].
-          1. Set |requestURL| to |request|'s [=request/url=].
-          1. If |options|.{{CacheQueryOptions/ignoreSearch}} is true, then:
-              1. Set |cachedURL|'s [=url/query=] to the empty string.
-              1. Set |requestURL|'s [=url/query=] to the empty string.
-          1. If |cachedURL| [=url/equals=] |requestURL| with the *exclude fragment flag* set, then:
-              1. Add a copy of |requestResponse|'s request to |requests|.
-              1. Add a copy of |requestResponse|'s response to |responses|.
-      1. Let |index| be zero.
-      1. For each |cachedResponse| in |responses|:
-          1. Let |cachedRequest| be |requests|[|index|].
-          1. Increment |index| by one.
-          1. If |cachedResponse|'s [=response/header list=] contains no <a>header</a> [=header/named=] \`<code>Vary</code>\`, or |options|.{{CacheQueryOptions/ignoreVary}} is true, then:
-              1. [=list/Append=] |cachedRequest|/|cachedResponse| to |resultList|.
-              1. [=Continue=].
-          1. Let |fieldValues| be the [=list=] containing the elements corresponding to the [=http/field-values=] of the <a>Vary</a> header.
-          1. Let |isMatched| be true.
-          1. For each |fieldValue| in |fieldValues|:
-              1. If |fieldValue| matches "`*`", or the [=combined value=] given |fieldValue| and |cachedRequest|'s [=request/header list=] does not match the [=combined value=] given |fieldValue| and |request|'s [=request/header list=], then:
-                  1. Set |isMatched| to false.
-                  1. [=Break=].
-          1. If |isMatched| is true, [=list/append=] |cachedRequest|/|cachedResponse| to |resultList|.
+          1. Let |cachedRequest| be |requestResponse|'s request.
+          1. Let |cachedResponse| be |requestResponse|'s response.
+          1. If [=Request Matches Cached Item=] with |requestQuery|, |cachedRequest|, |cachedResponse|, and |options| returns true, then add |cachedRequest|/|cachedResponse| to |resultList|.
```

Comment:
It seems this step lost semantics of using the "copy" of the request and the "copy" of the response?